packages/core/src/CheckBox/CheckBox.styles.tsx

```diff
@@ -19,6 +19,9 @@ export const { staticClasses, useClasses } = createClasses("HvCheckBox", {
   disabled: {
     cursor: "not-allowed",
     "& $label": { color: theme.colors.textDisabled, cursor: "not-allowed" },
+    "&:hover": {
+      backgroundColor: "transparent",
+    },
```

Comment:
isn't this redundant? we already have it in `BaseCheckBox` (with lots of specificity)